Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalMinBias_Output_cff.py

```diff
@@ -9,7 +9,10 @@
         SelectEvents = cms.vstring('pathALCARECOHcalCalMinBias')
     ),
     outputCommands = cms.untracked.vstring(
-	'keep *_gtDigisAlCaMB_*_*',
+        'keep *_hltTriggerSummaryAOD_*_*',
+        'keep *_TriggerResults_*_*',
+        'keep FEDRawDataCollection_rawDataCollector_*_*',
+        'keep FEDRawDataCollection_source_*_*',
```

Comment:
@bsunanda 
how much increase in size of event content and eventually dataset we expect by keeping all this information?